assignments/closure.js

```diff
@@ -13,9 +18,41 @@ const counter = () => {
 // newCounter(); // 1
 // newCounter(); // 2
 
+let count = 0;
+return function(){
+  count = count + 1;
+  return count;
+}
+};
+
+const newCounter = counter();
+console.log(newCounter());
+console.log(newCounter());
+
 // ==== Challenge 3: Create a counter function with an object that can increment and decrement ====
 const counterFactory = () => {
   // Return an object that has two methods called `increment` and `decrement`.
   // `increment` should increment a counter variable in closure scope and return it.
   // `decrement` should decrement the counter variable and return it.
 };
+
+const counterFactory = () => {
```

Comment:
The exercise already established the `counterFactory` function for you to work on in line 33. By declaring another `counterFactory` here, you will run into error, because you cannot declare two variables with the same name with `const`.